src/invidious/helpers/signatures.cr

```diff
@@ -30,7 +30,7 @@ struct DecryptFunction
 
       case op_body
       when "{a.reverse()"
-        operations[op_name] = ->(a : Array(String), b : Int32) { a.reverse }
+        operations[op_name] = ->(a : Array(String), _b : Int32) { a.reverse }
```

Comment:
What is the point/goal of that change?

Reply:
To mark `b` as unused. Proc notation doesn't seem to support using `_`.